### PR TITLE
Jia/chore: update width for heading

### PR DESCRIPTION
**Pull request by jia-deriv**
- update maxwidth 816px for block header
- fix fast payment block 'learn more' button size
image

### CODE REVIEW

**Review by hasan-deriv — DISMISSED** [review]

File: libs/blocks/src/lib/platform/card/index.tsx [hunk 1]
```diff
@@ -38,7 +38,11 @@ const PlatformBlockCard = ({
       )}
     >
       <FluidContainer className="flex flex-col gap-gap-3xl">
-        {header && <Heading.H2 className="text-center">{header}</Heading.H2>}
+        {header && (
+          <Heading.H2 className="mx-auto max-w-[816px] text-center">
```
Comment (fasihali-deriv) [inline libs/blocks/src/lib/platform/card/index.tsx:42]: Hi @jia-deriv just asking is that fine to add hardcoded value?

Comment (jia-deriv) [reply libs/blocks/src/lib/platform/card/index.tsx:42]: Designer fixed the width in figma to 816px for

Comment (hasan-deriv) [reply libs/blocks/src/lib/platform/card/index.tsx:42]: It's from designers. Since we are using max width so it's won't be a problem..

Comment (fasihali-deriv) [reply libs/blocks/src/lib/platform/card/index.tsx:42]: Thank you so much for the update

**Review by mohsen-deriv — APPROVED** [review]

**Pull request merged**